byzcoin/api.go

```diff
@@ -713,6 +713,16 @@ searchLatest:
 	return xerrors.New("didn't get the same blocks from everybody within 10 seconds")
 }
 
+// GetLatestTimestamp returns the timestamp of the latest known block
+func (c *Client) GetLatestTimestamp() (int64, error) {
+	header, err := decodeBlockHeader(c.getLatestKnownBlock())
```

Comment:
Hmm - we discussed this with @cgrigis , and I thought we'd take `time.Now` in the client - but I might have forgotten a good argument against this.

Reply:
When we discussed it with @ineiti, we mentioned the two possibilities, either taking the current time or the time of the latest block, and to examine how Ethereum did it.
Playing with [Remix](https://remix.ethereum.org/), it seems `now` returns the current machine time when using a JS VM, and the time of the last block when deployed on the Ropsten test chain.
I don't have a preference, so if you think using the current time is better I will update. :)

Reply:
I don't have any preference, my concern is about using `c.getLatestKnownBlock()` that uses `c.Latest` that corresponds to `c.Genesis` in the case no other blocks have been fetched by the clients, which can be dangerous. But if you think that's not a problem then let's keep it like that.

Reply:
You raise a good point. To avoid this, as well as @ineiti's point that ByzCoin has no guarantee that new blocks are created within a specific timeframe, I'll change it to take the machine's current time. It's not the exact same semantic as Ethereum, but it probably does not matter.